exporters/jaeger/src/TUDPTransport.cc

```diff
@@ -75,6 +75,10 @@ void TUDPTransport::close()
 
 uint32_t TUDPTransport::read(uint8_t *buf, uint32_t len)
 {
+  if (!server_addr_info_)
```

Comment:
Thanks for the catch. Could we also add internal log here?

Reply:
I was afraid that the log here would be too much if the the function is called often.
I add the log in a sec.

Reply:
Ok, perhaps add it right after the call to `getaddrinfo`.

Reply:
There is already a log there:

`[Error] File: exporters/jaeger/src/TUDPTransport.cc:57Jaeger Exporter: getaddrinfo failed with error: -2`

Shall I change the message?

Reply:
This current message is fine.